ui/components/app/wallet-overview/wallet-overview.js

```diff
@@ -6,15 +6,24 @@ import { toChecksumHexAddress } from '../../../../shared/modules/hexstring-utils
 import { getSelectedInternalAccount } from '../../../selectors';
 import { AddressCopyButton } from '../../multichain';
 import Box from '../../ui/box/box';
+import { IconName, Tag } from '../../component-library';
+import { Color, TextVariant } from '../../../helpers/constants/design-system';
+import { KeyringType } from '../../../../shared/constants/keyring';
+import { useI18nContext } from '../../../hooks/useI18nContext';
 
 const WalletOverview = ({
   balance,
   buttons,
   className,
   showAddress = false,
 }) => {
+  const t = useI18nContext();
   const selectedAccount = useSelector(getSelectedInternalAccount);
   const checksummedAddress = toChecksumHexAddress(selectedAccount.address);
+  const label = selectedAccount.metadata.snap?.name
+    ? `${selectedAccount.metadata.snap.name} (${t('beta')})`
+    : null;
+  const keyring = selectedAccount.metadata.keyring.type;
```

Comment:
Here for example, we can use the method `getLabel`

Reply:
Done!